lua/gatekeeperc.lua

```diff
@@ -8,40 +8,18 @@ struct gatekeeper_if {
 	char		**pci_addrs;
 	uint8_t		num_ports;
 	char		*name;
-	/*
```

Comment:
We should have a special comment here to help to debug:

`/* This struct has hidden fields. */`